tests/Tutor.Web.Tests/Integration/Instructors/InstructorTests.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Collections.Generic;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.DependencyInjection;
+using Shouldly;
+using Tutor.Web.Controllers.Domain.DTOs.Enrollment;
+using Xunit;
+
+namespace Tutor.Web.Tests.Integration.Instructors;
+
+[Collection("Sequential")]
+public class InstructorTests : BaseWebIntegrationTest
+{
+    public InstructorTests(TutorApplicationTestFactory<Startup> factory) : base(factory)
+    {
+    }
+    
+    [Fact]
+    public void Retrieves_courses()
+    {
+        using var scope = Factory.Services.CreateScope();
+        var controller = SetupInstructorController(scope, "-51");
+        var result = ((OkObjectResult)controller.GetCourses().Result)?.Value as List<CourseDto>;
+            
+        result.Count.ShouldBe(1);
+    }
+
+    [Fact]
```

Comment:
It would be better if both of these tests were theories, especially if you have two instructors in the test db scripts. You can use InlineData to pass the instructorId and expected course/group count.